packages/mobile/src/utils/currencies.ts

```diff
@@ -0,0 +1,65 @@
+export enum Currency {
+  Celo = 'Celo Gold',
+  Dollar = 'Celo Dollar',
+  Euro = 'Celo Euro',
+}
```

Comment:
I agree, removed everything we didn't really need. This may cause same backwards compatibility issues with payment requests though and it's hard to do something about it because there's always going to be a mismatch of versions.  Payment requests represent ~0.6% of total sends (450 total in last 30  days, [query](https://console.cloud.google.com/bigquery?sq=365081401101:10ddec9f32634d318c6dc239fbbbfd6a))

I think it's fine, but not sure. Wdyt?

Reply:
I'd prefer we kept backward compatibility.
We could use a cloud function to write translate the old/new field automatically.